eth/stagedsync/exec3.go

```diff
@@ -642,7 +642,7 @@ Loop:
 			return f(n)
 		}
 		blockContext := core.NewEVMBlockContext(header, getHashFn, engine, nil /* author */)
-
+		fmt.Println("B", blockNum, "txs", len(txs), "txNum", inputTxNum, "outputTxNum", outputTxNum.Load(), "blockComplete", blockComplete.Load())
```

Comment:
this print should be either made through logger with Debug or removed

Reply:
https://github.com/ledgerwatch/erigon/pull/10430